tools/seq_filter_by_id/seq_filter_by_id.py

```diff
@@ -214,7 +214,10 @@ def clean_name(name):
             if not line.strip():  # skip empty lines
                 continue
             if not line.startswith("#"):
-                file_ids.add(clean_name(line.rstrip("\n").split("\t")[col]))
+                current_id = line.rstrip("\n").split("\t")[col]
+                if current_id == "": #if for example, the column name is empty: typically R outputs
```

Comment:
I noticed later that style wise ``if not current_id:`` would be better. See https://www.python.org/dev/peps/pep-0008/#programming-recommendations

> For sequences, (strings, lists, tuples), use the fact that empty sequences are false.

Also, you only fixed the single-column branch (just above this is the code if the user wanted names from multiple columns).

I'll fix this in second commit...